engine-rest/engine-rest/src/main/java/org/operaton/bpm/engine/rest/impl/FetchAndLockHandlerImpl.java

```diff
@@ -357,6 +366,27 @@ protected void parseUniqueWorkerRequestParam(String uniqueWorkerRequestParam) {
     }
   }
 
+  protected void initializeQueue(int capacity) {
+    LOG.log(Level.FINEST, "Initializing queue with capacity [{0}]", capacity);
+    queue = new ArrayBlockingQueue<>(capacity);
+  }
+
+  private static int parseBlockingQueueCapacityParam(String queueSizeRequestParam) {
+    int capacity = DEFAULT_BLOCKING_QUEUE_CAPACITY;
+    if (queueSizeRequestParam != null) {
+      try {
+        final int parsedCapacity = Integer.parseInt(queueSizeRequestParam);
+        if (parsedCapacity <= 0) {
+          throw new IllegalArgumentException("Parameter " + BLOCKING_QUEUE_CAPACITY_PARAM_NAME + " has to be greater than zero");
+        }
+        capacity = parsedCapacity;
+      } catch (IllegalArgumentException e) {
```

Comment:
After this change, a `NumberParseException` thrown by `Integer#parseInt` is no longer handled, right?